### DISCUSSION TITLE
Already buy the profession version, BUT CANNOT USE，  does not receive any code

### DISCUSSION THREAD

**Discussion by yubai1111-000** [getlantern/lantern-forum-cn#637] [Q&A]
![fb1877536276d42f9fef1e8f7867b9a](https://github.com/getlantern/lantern-forum-cn/assets/173564474/13c47e27-eb23-450b-9a38-8f731bf59185)
used  yubaihuangyiwu@163.com the email buy the profession version but cannot use

**Comment by MaxMKMax** [comment]
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
@yubai1111-000 您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.
Would you mind directly send the issues to the link above attached with receipt, email and information? This approach might help to protect your privacy. Thanks.
